_includes/footer.html

```diff
@@ -11,13 +11,14 @@
                         </ul>
                     </div>
                     <div class="col-sm-3 col-xs-6">
-                        <h3>IRC Channel</h3>
+                        <h3>IRC Channels</h3>
```

Comment:
Not sure if this section should be kept because it just duplicates information from the hangouts page.

Reply:
What about changing to "IRC Main Channels"? o refer to the complete list in Hangouts? I think a short list of the main channels is useful. @rehrar thoughts on this?